examples/integration/test/fixtures/bwx.xcodeproj/project.pbxproj

```diff
@@ -9474,14 +9474,14 @@
 			isa = PBXNativeTarget;
 			buildConfigurationList = D08A821A46CE5B46FCADB0B1 /* Build configuration list for PBXNativeTarget "iOSApp" */;
 			buildPhases = (
-				936A0C13D99A28E18134B3D6 /* Create Compile Dependencies */,
-				738219E901C0E9AEF31E985E /* Create Link Dependencies */,
-				C0746DB917135C251EBC92A3 /* Sources */,
-				B6738331CF80F57EDBC55412 /* Resources */,
 				54E0573228E25BA06A17C8A4 /* Embed Frameworks */,
 				B728FD36D5E24A1628E973F5 /* Embed Watch Content */,
 				D3CC297B1790BC17C947B5D4 /* Embed App Extensions */,
 				A7E85E68A1BBCD520ACD5088 /* Embed App Clips */,
+				936A0C13D99A28E18134B3D6 /* Create Compile Dependencies */,
+				738219E901C0E9AEF31E985E /* Create Link Dependencies */,
+				C0746DB917135C251EBC92A3 /* Sources */,
+				B6738331CF80F57EDBC55412 /* Resources */,
```

Comment:
And we confirmed this doesn't create an issue, like compiling starting later for apps in BwX mode?

Reply:
Generated `Integration.xcodeproj` in both BwX & BwB mode and was able to build & run:
- iOS app ✅
- Watch app ✅
- macOS app ✅
Able to build extensions ✅